### PR TITLE
fix(rule): Remove unnecessary tests and other minor improvements

### PR DESCRIPTION
**Pull request by mauriciogardini**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pinterest/graphql-lint-rules",
-  "version": "0.3.0",
+  "version": "0.4.5",
```
Comment (jparise) [inline package.json:3]: I think you still want `0.3.0` here.

File: src/rules/compositeFieldsAreNullable.ts [hunk 2]
```diff
@@ -23,8 +23,7 @@ function isAllowedNonNullCompositeField(
   exceptions: Array<string>,
 ): boolean {
   // exceptions
-  const isException = exceptions.includes(type.name);
-  if (isException === true) {
+  if (exceptions.includes(type.name) === true) {
```
Comment (jparise) [inline src/rules/compositeFieldsAreNullable.ts:26]: Does this need `=== true`? It can only return a boolean, right?

File: package.json (hunk 1 above)
Comment (mauriciogardini) [reply package.json]: Yeah, I was changing that at the time you were reviewing this, I think.

File: src/rules/compositeFieldsAreNullable.ts (hunk 2 above)
Comment (mauriciogardini) [reply src/rules/compositeFieldsAreNullable.ts:26]: Yeah, you are right, this flew over my radar.

**Review by jparise — APPROVED** [review]

**Pull request merged**